### DISCUSSION TITLE
Margin vs without margin emission factor

### DISCUSSION THREAD

**Discussion by Anush360** [USEPA/supply-chain-factors#82] [Q&A]
Hi! According to the definition of supply chain emission factor **without margin**, it includes activities from production of goods up until they leave the factory gate (cradle-to-factory gate) and **with margin** includes total emissions from product's creation to its availability in store shelves (cradle-to-shelf). For example I am a retailer of shoes then-
1. Should I use EFs with margin or without margin if transportation of product is already included in scope 3 category 4 and operation emission from buildings is accounted in scope 1/2?
2. Which USEEIO/NAICS category emissions should I use- Clothing '31-33: Manufacturing/3151: Apparel Knitting Mills' or Clothing and clothing accessories stores '44-45: Retail Trade/4481: Clothing Stores'?

**Comment by bl-young** [comment]
1. Given the example you describe, the factors without margins seem more appropriate (so as not to include operation emissions from retail and transport).
2. Use the manufacturing sector (e.g. 3151) and not the retailing sector. The retailing sectors are margin sectors and do not include the emissions from the manufacture of the goods itself.

**Comment by Brian-Solomon** [comment]
@bl-young, Category 4 emissions would represent emissions from shelf _or_ factory gate to the purchaser's facility, and there is no way that the USEEIO can account for this component of the supply chain (i.e., "shelf to buyer facility") unless there are cases where shipping (to the final customer) is included in the purchaser price. 

Per the July 2020 guidance document, the margins of the supply chain emission factors represent:
"_Margins of Supply Chain Emission Factors: emissions associated with factory gate to shelf, which includes emissions from transportation, wholesale and retail as well as adjustments for price markups_" (page xii)

"_The associated margin emission factors add the emissions associated with transporting, storing and selling the commodity._" (page 3)

https://cfpub.epa.gov/si/si_public_file_download.cfm?p_download_id=540798&Lab=CESER

It seems the more crucial question for this shoe retailer would be:
"Do you purchase shoes direct-from-manufacturer or do you buy from a wholesaler/distributer?" 
If the former, then SEF only, if the latter, then SEF+MEF.

Does this sound right?

**Comment by TimCroker** [comment]
See also my question about which prices to use:
https://github.com/USEPA/supply-chain-factors/discussions/92
